src/setup/bootstrap/createStashRecords.ts

```diff
@@ -77,6 +78,34 @@ export const createSampleStashRecords = async ({
     ],
   } satisfies TransactionSetDestinations);
 
+  await saveErrorDestinations("destinations|errors|file_error", {
+    $schema:
+      "https://raw.githubusercontent.com/Stedi-Demos/bootstrap/main/src/schemas/error-destinations.json",
+    description: "Send file errors to webhook",
+    destinations: [
+      {
+        destination: {
+          type: "webhook",
+          url: requiredEnvVar("DESTINATION_WEBHOOK_URL"),
+        },
+      },
+    ],
+  });
+
+  await saveErrorDestinations("destinations|errors|execution", {
+    $schema:
+      "https://raw.githubusercontent.com/Stedi-Demos/bootstrap/main/src/schemas/error-destinations.json",
+    description: "Send function execution errors to webhook",
```

Comment:
I would label this as "Send delivery failures to webhook"  - its not tied to the DLQ so "function execution errors" is confusing?

Reply:
@BDQ: apologies I hadn't followed up. this PR got lost in the shuffle... I think `function execution errors` is more accurate as it encompasses things beyond just delivery failures. these destinations are essentially replacing the old `failure.json` files for failed execution tracking.